core/src/main/java/io/questdb/griffin/engine/functions/bind/StrBindVariable.java

```diff
@@ -192,9 +187,9 @@ public void setValue(float value) {
         isNull = Numbers.isNull(value);
         if (!isNull) {
             utf16Sink.clear();
-            utf16Sink.put(value, floatScale);
+            utf16Sink.put(value);
```

Comment:
Double scale may be too high for a `float` value, so if there is a later float parsing, it might error after this change.

Reply:
See previous comment, PR adds `CharSink::put(float)` and also created `Numbers::MAX_FLOAT_SCALE = 10` which is used for float, and kept `Numbers::MAX_DOUBLE_SCALE = 19` for double.
I think max precision should be ok too.